Core/src/org/sleuthkit/autopsy/centralrepository/persona/PersonaDetailsPanel.java

```diff
@@ -535,14 +532,13 @@ private void nameFieldActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIR
         "PersonaDetailsPanel_load_exception_msg=Failed to load persona",})
     private void loadPersona(Component parent, Persona persona) {
         String name;
-        Collection<CentralRepoAccount> accounts;
+        Collection<PersonaAccount> accounts;
         Collection<PersonaMetadata> metadata;
         Collection<PersonaAlias> aliases;
         Collection<CorrelationCase> cases;
         try {
             name = persona.getName();
-            accounts = persona.getPersonaAccounts().stream().map(PersonaAccount::getAccount)
-                    .collect(Collectors.toList());
+            accounts = persona.getPersonaAccounts();
```

Comment:
Based on Brian's response, I think we should also show the persona creation date, examiner, and comment.

Also the Persona Creation dialog should have a field for the user to enter a comment.